### ISSUE TITLE
Feed Value Error

### ISSUE THREAD

**Issue by akjadhav** [akjadhav/CNNDownSyndrome#1]
`import PIL
from PIL import Image
import numpy as np
from os import listdir
import skimage
from skimage import transform
from skimage import data
from skimage.color import rgb2gray
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.metrics import classification_report, confusion_matrix
import matplotlib.pyplot as plt
import matplotlib.image
from sklearn.utils import shuffle
import tensorflow as tf

def CNN_Preprocess():
    normal_photo_base_path = r'Z:\Share\DownSyndrome\All Photo\TEST\Normal'
    down_syndrome_photo_base_path = r'Z:\Share\DownSyndrome\All Photo\TEST\Down'
    williams_syndrome_photo_base_path = r'Z:\Share\DownSyndrome\All Photo\TEST\Williams'

    normal_filenames = []
    for f in listdir(normal_photo_base_path):
        normal_filenames.append(normal_photo_base_path + '\\' + f)

    down_filenames = []
    for f in listdir(down_syndrome_photo_base_path):
        down_filenames.append(down_syndrome_photo_base_path + '\\' + f)

    williams_filenames = []
    for f in listdir(williams_syndrome_photo_base_path):
        williams_filenames.append(williams_syndrome_photo_base_path + '\\' + f)

    image_filenames = normal_filenames + down_filenames + williams_filenames

    normal_label = 0
    down_label = 1
    williams_label = 2

    global images
    images = []

    normal_labels = [normal_label] * len(normal_filenames)
    down_labels = [down_label] * len(down_filenames)
    williams_labels = [williams_label] * len(williams_filenames)

    labels = normal_labels + down_labels + williams_labels
    labels = np.array(labels)

    for image in image_filenames:
        images.append(skimage.data.load(image))

    images = np.array(images, ndmin = 3)
    #images = images.flatten()
    
    return images, labels

images, labels = CNN_Preprocess()

images = shuffle(images, random_state = 101)
labels = shuffle(labels, random_state = 101)

X_train, X_test, y_train, y_test = train_test_split(images, labels, test_size=0.25, random_state=101)

def init_weights(shape):
    init_random_dist = tf.truncated_normal(shape, stddev = 0.1)
    return tf.Variable(init_random_dist)

def init_bias(shape):
    init_bias_vals = tf.constant(0.1, shape = shape)
    return tf.Variable(init_bias_vals)

def conv2d(x, W):
    return tf.nn.conv2d(x, W, strides=[1, 1, 1, 1], padding='SAME')

def max_pool_2by2(x):
    return tf.nn.max_pool(x, ksize=[1, 2, 2, 1],
                          strides=[1, 2, 2, 1], padding='SAME')

def convolutional_layer(input_x, shape):
    W = init_weights(shape)
    b = init_bias([shape[3]])
    return tf.nn.relu(conv2d(input_x, W) + b)

def normal_full_layer(input_layer, size):
    input_size = int(input_layer.get_shape()[1])
    W = init_weights([input_size, size])
    b = init_bias([size])
    return tf.matmul(input_layer, W) + b

x = tf.placeholder(tf.float32,shape=[None,244,244,3])
y_true = tf.placeholder(tf.float32,shape=[None,1])

x_image = tf.reshape(x,[-1,244,244,1])

convo_1 = convolutional_layer(x_image,shape=[6,6,1,32])
convo_1_pooling = max_pool_2by2(convo_1)

convo_2 = convolutional_layer(convo_1_pooling,shape=[6,6,32,64])
convo_2_pooling = max_pool_2by2(convo_2)

convo_2_flat = tf.reshape(convo_2_pooling,[-1,61*61*64])
full_layer_one = tf.nn.relu(normal_full_layer(convo_2_flat,1024))

hold_prob = tf.placeholder(tf.float32)
full_one_dropout = tf.nn.dropout(full_layer_one,keep_prob=hold_prob)

y_pred = normal_full_layer(full_one_dropout,10)

cross_entropy = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(labels=y_true,logits=y_pred))

optimizer = tf.train.AdamOptimizer(learning_rate=0.0001)
train = optimizer.minimize(cross_entropy)

init = tf.global_variables_initializer()

def next_batch(num, data, labels):
    '''
    Return a total of `num` random samples and labels.
    '''
    idx = np.arange(0 , len(data))
    np.random.shuffle(idx)
    idx = idx[:num]
    data_shuffle = [data[ i] for i in idx]
    labels_shuffle = [labels[ i] for i in idx]

    return np.asarray(data_shuffle), np.asarray(labels_shuffle)

steps = 3

with tf.Session() as sess:
    sess.run(init)

    for i in range(steps):
        print(i)

        batch_x, batch_y = next_batch(5, X_train, y_train)

        sess.run(train, feed_dict={x: batch_x, y_true: batch_y, hold_prob: 0.5})

        if True:
            print('Currently on step {}'.format(i))
            print('Accuracy is:')
            # Test the Train Model
            matches = tf.equal(tf.argmax(y_pred, 1), tf.argmax(y_true, 1))

            acc = tf.reduce_mean(tf.cast(matches, tf.float32))

            print(sess.run(acc, feed_dict={x: X_test, y_true: y_test, hold_prob: 1.0}))
            print('\n')
`

My code is above. I have tried everything but I get a type error that says...

"Cannot feed value of shape (5,) for Tensor 'Placeholder_22:0', which has shape '(?, ?)'"
